bindings/python/scripts/dlite-validate

```diff
@@ -140,6 +140,22 @@ def main():
     for inst in insts:
         if inst.is_meta:
             check_dimensions(args.url, inst)
+            # Check that datamodels for properties that refer to
+            # a specific datamodel can be loaded.
+            for prop in inst.properties['properties']:
+                if prop.type == 'ref':
+                    if prop.ref == 'ref':
+                        print(f'Property {prop.name} is a reference, but a '
+                                'uri is not given. Check that the property '
+                                '$ref is given, the $ is needed.')
+                        sys.exit(1)
+                    # Add the path of the original url to storage_path
+                    # since it is likely that referenced data models are 
+                    # in the same folder.
+                    dlite.storage_path.append(Path(args.url).resolve().parent)
```

Comment:
This should work great in the common case that `args.url` is a file path.

However, although not doing any harm, this may not correctly add the storage to `dlite.storage_path` when `args.url` is, say a mongodb storage. In this case, the user will have to use the `--storage-path` option. Maybe the comment (or the 'help' for `--storage-path` option) should be extended with a note about this case?

Reply:
But in that case you have to give the storage path as argument anyway, no? I updated the doc-string to the --storage-path argument

Reply:
I see. Great